pkg/sqlite3/sqlite3.go

```diff
@@ -62,17 +62,19 @@ func (db *SQLite3) Read() <-chan dbi.Entry {
 			}
 		}
 
-		for rows.Next() {
-			var blob string
-			if err := rows.Scan(&blob); err != nil {
-				entries <- dbi.Entry{
-					Err: xerrors.Errorf("failed to Scan Row: %w", err),
+		if rows != nil { // nil if db.Query failed
```

Comment:
Instead of indenting the whole block, would it make sense to just check for rows == nil and return?

Reply:
Done.